website/docs/export.md

```diff
@@ -209,6 +211,106 @@ Dapr: https://dapr.io/
+
+1. Build `fake-reader` image from [gatekeeper/test/export/fake-reader](https://github.com/open-policy-agent/gatekeeper/tree/master/test/export/fake-reader)
+
+    ```bash
+    docker buildx build -t <your_img_name:tag> --load -f test/export/fake-reader/Dockerfile test/export/fake-reader
+    ```
+
+    **Note:** Make sure the fake-reader image is available in your preferred registry or cluster.
+
+2. Create `values.yaml` with the following variables.
+
+    ```yaml
+    audit: 
+      exportVolume: 
+        <your-volume> 
+      exportVolumeMount: 
+        <your-volume-mount-path>
+      exportSidecar: 
+        <your-side-car>
+    ```
+
+    Here is the default `values.yaml` that you can use.
+
+    ```yaml
+    audit: 
+      exportVolume: 
+        name: tmp-violations 
+        emptyDir: {} 
+      exportVolumeMount: 
+        path: /tmp/violations 
+      exportSidecar: 
+        name: go-sub 
+        image: fake-reader:latest 
+        imagePullPolicy: Always 
+        securityContext: 
+          allowPrivilegeEscalation: false 
+          capabilities: 
+            drop: 
+            - ALL 
+          readOnlyRootFilesystem: true 
+          runAsGroup: 999 
+          runAsNonRoot: true 
+          runAsUser: 1000 
+          seccompProfile: 
+            type: RuntimeDefault 
+        volumeMounts: 
+        - mountPath: /tmp/violations 
+          name: tmp-violations
+    ```
+
+3. Deploy Gatekeeper charts with `values.yaml`.
+
+    ```shell
+    helm upgrade --install gatekeeper gatekeeper/gatekeeper --namespace gatekeeper-system \
+    ...
+    --set enableViolationExport=true \
+    --set audit.connection=audit-connection \
+    --set audit.channel=audit-channel \
+    --values /path/to/values.yaml
+    ```
+
+4. Create connection config to establish a connection.
+
+    ```shell
+    kubectl apply -f - <<EOF
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: audit
+      namespace: gatekeeper-system
+    data:
+      driver: "disk"
+      config: |
+        {
+          "path": "/tmp/violations",
+          "maxAuditResults": 3
+        }
+    EOF
+    ```
+
+    **Note:** Name of the connection configMap must match the value of `--audit-connection` for it to be used by audit to export violation. At the moment, only one connection config can exists for audit.
+
+5. Create the constraint templates and constraints, and make sure audit ran by checking constraints. If constraint status is updated with information such as `auditTimeStamp` or `totalViolations`, then audit has ran at least once. Additionally, populated `TOTAL-VIOLATIONS` field for all constraints while listing constraints also indicates that audit has ran at least once.
+
+    ```log
+    kubectl get constraint
+    NAME                 ENFORCEMENT-ACTION   TOTAL-VIOLATIONS
+    pod-must-have-test                        0
+    ```
+
+6. Finally, check the sidecar reader logs to see the violations written.
+
+    ```log
```

Comment:
Getting this error after following the steps:
```console
kubectl logs -l gatekeeper.sh/operation=audit -c go-sub -n gatekeeper-system

2025/03/26 02:00:24 Error finding latest file no files found in directory: /tmp/violations/
2025/03/26 02:00:29 available files []
2025/03/26 02:00:29 reading from
```

Reply:
Ahh good catch. The configMap name needs to be `audit-connection` to resolve this error. I will update the docs.

Reply:
Docs should be fixed.

Reply:
confirming the latest changes fixed this issue